requirements.txt

```diff
@@ -10,5 +10,6 @@ boto==2.40.0
 apache-libcloud==0.20.1
 sphinx==1.4.4
 jsonschema==2.5.1
-libvirt-python==2.1.0
```

Comment:
@herlo : Readthedocs is fail to build linchpin docs when libvirt-python is mentioned in requirement.txt , I removed the package from requirement.txt in oder to make successful build in read the docs .